content/docs/command-reference/move.md

```diff
@@ -69,6 +71,46 @@ outs:
     path: other.csv
 ```
 
+### Manually renaming stage outputs
+
+In this example, let's rename stage output `keras.h5` to `model.h5`:
+
+```dvc
+$ cat dvc.yaml
+stages:
+  train:
+    cmd: python train.py
+    ...
+    outs:
+    - keras.h5
+```
+
+First we need to change the name of model file in code (`train.py`). Next we
+change the output name in the `train` stage of `dvc.yaml`. After this we change
+`/keras.h5` to `/model.h5` in the `.gitignore` file. Lastly, we rename the
+existing model file:
+
+```dvc
+$ mv keras.h5 model.h5
+```
+
+In the final step we run `dvc commit` with `-f` option to force save the
+changes:
```

Comment:
Grammar

Reply:
Maybe explain a but more what commit -f does. Very briefly

Reply:
You know what, I guess there's no need to explain commit here. Users can click on the link if they're interested.